plugins/auth-backend/src/service/router.ts

```diff
@@ -68,7 +68,8 @@ export async function createRouter(
   if (secret) {
     router.use(cookieParser(secret));
     // TODO: Configure the server-side session storage.  The default MemoryStore is not designed for production
-    router.use(session({ secret, saveUninitialized: false, resave: false }));
+    const enforceCookieSSL = process.env.NODE_ENV === 'production';
```

Comment:
Could we perhaps base it on the external URL of the backend instead? the `authUrl` in this code. I think that'd make sure we don't ruin things for people that really wanna run HTTP in production or that sorta thing :grin:

Reply:
changed condition to use authUrl.startWith('https')